addons/t9n/static/src/core/resource_page.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<templates>
+
+    <t t-inherit="web.FormView" t-inherit-mode="primary" t-name="t9n.ResourcePage">
+        <xpath expr="//Layout" position="replace">
+
+            <div class="row overflow-auto">
+                 <div class="col-4 px-0">
+                    <MessageList onChange.bind="onMessageClick" resource_id="this.model.root.evalContext.context.active_id"/>
```

Comment:
@oomsveta 
I was happy with my approach but now after reading it again I think I am over-engineering that. I think there is no need to create a separate component `MessageList` and then bind a function from the parent to it. Also, `this.fetchMessages()` is called twice with this approach. Once when clicking on any message in `MessageList` component and once in the to render the editor. 

What do you think?